doc/src/learn/sponsored-transactions.md

```diff
@@ -0,0 +1,161 @@
+ * [TransferSui](https://docs.sui.io/sui-jsonrpc#sui_transferSui)
+ * [PaySui](https://docs.sui.io/sui-jsonrpc#sui_paySui)
+ * [PayAllSui](https://docs.sui.io/sui-jsonrpc#sui_payAllSui)
+
+ Also note that you can’t use Sponsored Transaction with `BatchTransaction`, which will be deprecated in a future release.
+
+
+## Potential Risks Using Sponsored Transactions
+
+The most significant potential risk when using sponsored transactions is [equivocation](../learn/sui-glossary#equivocation). In some cases under certain conditions, a sponsored transaction can result in all associated owned objects, including gas in a locked state when examined by Sui validators. To avoid double spending, validators lock objects as they validate transactions. An equivocation occurs when an owned object’s pair (`ObjectID`, `SequenceNumber`) is concurrently used in multiple non-finalized transactions.
+
+To equivocate, either the user or the sponsor signs and submits another transaction that attempts to manipulate an owned object in the original transaction. Because only the object owner can use an owned object, only the user and sponsor can cause this condition.
+
+## Create a user-initiated sponsored transaction
+
+A user-initiated sponsored transaction involves the following steps:
+
+ 1. A user initializes a `GasLessTransactionData` transaction.
+ 1. The user sends `GasLessTransactionData` to the sponsor.
+ 1. The sponsor validates the transaction, constructs `TransactionData` with gas fees, and then signs `TransactionData`.
+ 1. The sponsor sends the signed `TransactionData` and the sponsor `Signature` back to the user.
+ 1. The user verifies and then signs `TransactionData` and sends the dual-signed transaction to Sui network through a Full node or the sponsor.
+
+### GasLessTransactionData
+
+GasLessTransactionData is basically `TransactionData` less `GasData`. It is not a sui-core data structure, but it is only an interface between user and sponsor.
+
+The following example constructs a GasLessTransactionData  object.
+
+```rust
+pub struct GasLessTransactionData {
+    pub kind: TransactionKind,
+    sender: SuiAddress,
+    …
+}
+```
+
+## Create a sponsor-initiated sponsored transaction  
+
+A sponsor-initiated sponsored transaction involves the following steps:
+ 1. A sponsor constructs a TransactionData object that contains the transaction details and associated gas fee data. The sponsor signs it to generate a Signature  before sending it to a user. 
+ You can send the unsigned TransactionData via email, SMS, or an application interface.
+ 1. The user checks the transaction and signs it to generate the second signature for the transaction.
+ 1. The user submits the dual-signed transaction to a Sui Full node or sponsor to execute it.
```

Comment:
we should codify `TransactionData`, `signature` etc as we do in other two cases?

Reply:
yes, thanks for catching. Added to a few other places as well